### PR TITLE
feat: add environment variables step

### PR DESCRIPTION
**Pull request by Pesteves2002**

### CODE REVIEW

File: tutorial/step3.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+```
+git config --global user.name 'Test user'
+git config --global user.email '<>'
+
+git commit -am "feat: add environment variables"
+
+git push piku master
+```{{exec}}
+
+Your changes will be deployed and you should be able to see that the service is now listening on port 2482 [http://localhost:2482]({{TRAFFIC_HOST1_2482}}).
```
Comment (Pesteves2002) [inline tutorial/step3.md]: @randomicecube look at this

**Comment by randomicecube** [comment]
@Pesteves2002 add a `verify.sh` to step3 which checks for the port

**Comment by randomicecube** [comment]
@Pesteves2002 I think that we need to set the port up in the deploy step, so it doesn't make sense to repeat it here; instead, change the source code to make use of the `TOOL` env var, such that it's displayed in the website

**Comment by randomicecube** [comment]
Remove adding a name/email to the git config, since this is done in the prev step

**Pull request merged**